### DISCUSSION TITLE
Reporting Types and Rules as Open Source

### DISCUSSION THREAD

**Discussion by tomhealey-icma** [finos/common-domain-model#2378] [Q&A] [answered]
Requesting a clarification on use of Rosetta Reporting data types, rules and runtime module.

**Comment by lolabeis** [comment, marked as answer]
This is documented here (Digital Regulatory Reporting documentation):
https://drr.docs.rosetta-technology.io

Clarification on how to use the runtime module can be found here:
https://drr.docs.rosetta-technology.io/source/implementation.html#transform
(Note that this piece of the DRR infra is in the process of being upgraded, so this part of the doc will be changing very soon).
